language-extensions/R/build/windows/restore-packages.cmd

```diff
@@ -21,7 +21,7 @@ IF "%R_HOME%" == "" (
 	REM
 	SET R_HOME=%DEFAULT_R_HOME%
 	MKDIR %DEFAULT_R_HOME%
-	powershell -Command "Invoke-WebRequest https://cran.microsoft.com/bin/windows/base/old/%R_VERSION%/%R_INSTALLER%  -OutFile %DEFAULT_R_HOME%\%R_INSTALLER%"
+	powershell -Command "Invoke-WebRequest https://cran.r-project.org/bin/windows/base/old/%R_VERSION%/%R_INSTALLER%  -OutFile %DEFAULT_R_HOME%\%R_INSTALLER%"
```

Comment:
need to have `/windows/base/old` with "old" because this may download R3.5.2? and R3.5.2 is legacy now?

Reply:
According to my understanding yes, since https://cran.r-project.org/bin/windows/base/old/ contains previous binary releases of R for Windows.